src/client/Client.cc

```diff
@@ -7723,12 +7750,18 @@ int Client::fill_stat(Inode *in, struct stat *st, frag_info_t *dirstat, nest_inf
       st->st_size = in->rstat.rbytes;
     else
       st->st_size = in->dirstat.size();
+    #ifndef _WIN32
     st->st_blocks = 1;
+    #endif
   } else {
     st->st_size = in->size;
+    #ifndef _WIN32
     st->st_blocks = (in->size + 511) >> 9;
+    #endif
   }
+  #ifndef _WIN32
   st->st_blksize = std::max<uint32_t>(in->layout.stripe_unit, 4096);
+  #endif
```

Comment:
Why do you need to exclude windows on these values? Does their stat struct not contain those fields?
A comment explaining that might be nice.

Reply:
Indeed, an inline comment would help. The Windows ``stat`` structure has just a small subset of the Linux fields: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/stat-functions?view=msvc-160#generic-text-routine-mappings